src/libs/data-utils.js

```diff
@@ -380,11 +385,9 @@ export const EntityUploader = class EntityUploader extends Component {
               ...Utils.newTabLinkProps
             }, ['Learn more ', icon('pop-out', { size: 12 })])
           ]),
-          isInvalid && div({
-            style: { color: colors.danger(), fontWeight: 'bold', fontSize: 12, marginTop: '0.5rem' }
-          },
-          ['Invalid format: Data does not start with entity or membership definition.']
-          )
+          div({ style: errorTextStyle },
+            [(isInvalid && 'Invalid format: Data does not start with entity or membership definition.') ||
```

Comment:
Wouldn't we potentially want both? What if I've pasted something incorrect but then I try to edit it in place?

Reply:
Both as in pasting and text editing? The decided on approach is to only allow copy&pasting right now in an effort to add the minimal requested feature and also avoiding user error particularly with the tab separation. If in the future there is a need/desire to add editing that is fine but will require figuring out how to deal with the tab separation and other user editing pitfalls here appropriately.